build_data/context.xml

```diff
@@ -20,7 +20,7 @@
       auth="Container"
       type="javax.sql.DataSource"
       driverClassName="org.postgresql.Driver"
-      url="jdbc:postgresql://${HOST}:${POSTGRES_PORT}/${POSTGRES_DB}?${SSL_PARAMETERS}"
+      url="jdbc:postgresql://${HOST}:5432/${POSTGRES_DB}?${SSL_PARAMETERS}"
```

Comment:
This should remain ${POSTGRES_PORT}, You can link to a DB that is not running in docker and it could have a none standard port